pyfa/modules/setup_shell_commands.py

```diff
@@ -9,20 +9,23 @@
 import os
 
 main_path = os.path.dirname(__file__)
-executor_file = os.path.join(main_path, 'bash_executor.sh')
-
+executor_file = os.path.join(main_path, '../../bash_executor.sh')
```

Comment:
Better to use something like this:
```
from pathlib import Path
lib_folder = Path(__file__).resolve().parents[2]
```


*This construct the path 2-levels up* 
Because the backslash will not work on a windows system (to make it future proof).